lib/smart_answer_flows/coronavirus-employee-risk-assessment.rb

```diff
@@ -66,15 +49,23 @@ def define
             question :is_your_workplace_an_exception?
           when "auction_house"
             question :is_your_workplace_an_auction_house?
-          when
-            "salon_parlour",
-            "nightclubs_or_gambling",
-            "indoor_recreation",
-            "funfair",
-            "indoor_visitor_centres"
-            outcome :workplace_should_be_closed
+          when "other"
+            question :can_work_from_home?
           else
-            question :are_you_shielding?
+            question :is_your_employer_asking_you_to_work?
+          end
+        end
+      end
+
+      multiple_choice :is_your_workplace_an_auction_house? do
```

Comment:
Can we make this is a distinction that this is specifically about "livestock" auction houses? I was just confused - because the previous question already asks if you are work at an auction house.

Reply:
Sorry I realise the naming wasn't apart of this PR - so not a blocker for this. But might be good to change in the future.

Reply:
This is a good point - cc @DGarland5